src/components/HomePage/Banner.js

```diff
@@ -90,11 +117,19 @@ function renderInfo(classes) {
 					width: 150,
 					margin: 20
 				}} />
-			<Typography variant='h5' className={classes.text} component='h3'>
-				<time dateTime={hothStart.toISOString()}>
-					Feb 23<sup>rd</sup>, 2021
-				</time>
-			</Typography>
+			<Box className={classes.horizontalBox}>
+				<Tooltip title={eventCrossesDate ?
+					`It looks like HOTH crosses between dates in your timezone! (${tz})` :
+					''}
+				placement='top'
+				arrow='true'>
+					<Typography variant='h5' className={classes.text} component='h3'>
+						<time dateTime={hothStart.toISOString()}>
+							{month} {startDay}<sup>{ordinal(startDay)}</sup>{endDayElement}, 2021
+						</time>
+					</Typography>
+				</Tooltip>
```

Comment:
I thought I'd throw in the tooltip and see what you all thought. It provides a bit more explanation to the user.

Reply:
I like the tooltip!